deploy/ft/ft-create.js

```diff
@@ -13,6 +14,13 @@ async function start() {
 	let near = await initiateNearConnection(NETWORK_ID);
 	const fundingAccount = await near.account(FUNDING_ACCOUNT_ID);
 
+	//get amount to transfer and see if owner has enough balance to fund drop
+	let amountToTransfer = new BN(FT_DATA.balancePerUse).mul(new BN(NUM_KEYS * DROP_CONFIG.usesPerKey))
+	console.log('amountToTransfer: ', amountToTransfer.toString());	
```

Comment:
```suggestion
	console.log('amountToTransfer: ', amountToTransfer);	
```

Reply:
same as other suggestion; used `amountToTransfer` for comparison and used .toString() anywhere a string was needed